.cargo/config.toml

```diff
@@ -0,0 +1,48 @@
+[build]
```

Comment:
Is it better to configure this with rustflags or in the Cargo.toml? I've used the Cargo.toml option to warn on pedantic as well as disabling a couple temporarily. 

I tried enabling the nursery lints in another project and ended up turning it off because a lot ended up just being annoying. I assume a lot of these that have been manually allowed are probably nursery too?

Reply:
I don't know if there is an accepted/idiomatic answer.
IMO `Cargo.toml` is included in the library publishing process and users read it, I feel like the crate's clippy config isn't of users concern (as in "users don't care", not "should be hidden") so we shouldn't bother them with it 🤔